crossbuilder

```diff
@@ -507,14 +507,27 @@ apply_build_profiles_to_debian_control () {
     done
 }
 
+# Accepts file paths. Print the first file that exists, or nothing if none exists.
+first_existing_file () {
+  while [ -n "$1" ]; do
+    if [ -e "$1" ]; then
+      echo "$1"
+      return
+    fi
+
+    shift
+  done
+}
+
 ensure_upstream_tarball () {
-    if [ -f ubports.source_location ]; then
+    source_location_file=$(first_existing_file ubports.source_location debian/ubports.source_location)
```

Comment:
I guess these should be reversed like in other PR, so that `debian/` version is checked for first.

Reply:
Also fixed. Sorry for the delay.